frontend/src/components/organism/Projections.tsx

```diff
@@ -98,8 +109,14 @@ export default styled(Projections)`
     display: grid;
     grid-template-columns: repeat(5, 1fr);
 
+    padding-left: 0.5em;
+    padding-right: 0.5em;
```

Comment:
```suggestion
  padding: 0 0.5em;
```

Presuming there is no other padding applied.

Reply:
I think that there is already some default padding applied, so doing this will remove that.